plugins/apple/swift/v1.26.0/Dockerfile

```diff
@@ -1,14 +1,19 @@
 # syntax=docker/dockerfile:1.7
-FROM swift:5.10.0-jammy AS build
+FROM swift:5.10.1-bookworm AS build
 
-COPY extramoduleimports.patch /app/extramoduleimports.patch
+RUN apt-get update \
+ && apt-get install -y libstdc++-12-dev
```

Comment:
Why needed to add `libstdc++-12-dev` ?

Reply:
Without it you get a linker failure creating the static binary.